package.json

```diff
@@ -21,23 +21,26 @@
     "classnames": "^2.2.1"
   },
   "devDependencies": {
-    "ava": "^0.15.0",
+    "ava": "^0.19.1",
     "babel-cli": "^6.6.5",
     "babel-core": "^6.7.2",
+    "babel-eslint": "^7.2.1",
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-react": "^6.5.0",
     "babel-preset-stage-1": "^6.5.0",
-    "eslint": "^2.4.0",
-    "eslint-config-i-am-meticulous": "^3.0.0",
-    "eslint-plugin-react": "^3.0.0",
+    "eslint": "^3.19.0",
+    "eslint-config-i-am-meticulous": "^6.0.0",
+    "eslint-plugin-import": "^2.2.0",
+    "eslint-plugin-react": "^6.10.3",
     "npmpub": "^3.1.0",
+    "prop-types": "^15.5.7",
     "react": "^15.2.0",
     "react-addons-test-utils": "^15.2.0",
     "react-dom": "^15.2.0",
     "rimraf": "^2.4.4"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0-rc.1"
+    "react": "^0.14.0 || ^15.0.0"
```

Comment:
It seems like that part is unnecessary. That semver range would result in the latest 15 release, no?

![screenshot 2017-04-12 09 44 28](https://cloud.githubusercontent.com/assets/109114/24961388/94ffba74-1f66-11e7-9214-417b14715140.png)

Maybe I am unclear of what you are doing here. I will add it back for you.